### PR TITLE
feat(api): add p card client

### PR DESCRIPTION
**Pull request by thorkellmani**
## What

* Add a p card client that connects to Syslumenn

## Why

Add support for the display of p cards in licenses

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by busla — APPROVED** [review]
LGTM

**Review by osk — APPROVED** [review]
👌

File: libs/clients/p-card/project.json
Comment (saevarma) [inline libs/clients/p-card/project.json:1]: #12379 is changing the target names for `schemas/`.
Just wanted to comment, but this PR will probably be merged first, so 12379 will need to fix that.

**Review by saevarma — APPROVED** [review]
LGTM

**Pull request merged**